src/lib/api/api.go

```diff
@@ -24,12 +37,120 @@ func (a *API) Router() *mux.Router {
 	return a.router
 }
 
+func (a *API) UpdateConf(conf *config.Config) {
+	d := make(map[string][]config.API)
+	for _, api := range conf.APIs {
+		d[api.Domain] = append(d[api.Domain], api)
+	}
+	a.mutex.Lock()
+	a.domainMap = d
+	log.Info("msg", "Updated API domain map")
+	a.mutex.Unlock()
+}
+
 func (a *API) RegisterRoutes() {
 	a.router.HandleFunc("/api/v1/reload", a.Reload)
+	a.router.HandleFunc("/get-machines", a.getMachines)
+	a.router.HandleFunc("/get-domain-entities", a.getDomainEntity)
```

Comment:
the route need to be prefixed with /api/v1/

Reply:
Okay, will do the changes!